### PR TITLE
[No Ticket] git secrets and readme updates

### PR DESCRIPTION
**Pull request by s-rubenstein**
Ticket: [No Ticket]
I found that subject specified a user in the readme, but above it said user or group. I just edited to make consistent. While doing this, I found that I got a warning from hooks. It looked like it didn't actually have execute permissions on some of our gitsecrets stuff. In digging in, we found that the hooks directory didn't actually do anything, and minnie-kenny was offering our protection. The same git-secrets hooks directory is in other repos, but minnie-kenny is not there, so we should probably do an audit of our repos.

---

**PR checklist**

- [x] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,13 @@
 ## In a nutshell
 The crux of IAM in Sam is a policy. A policy says **who** can **do what** to a **thing**. More technically the who is called a **subject** and can be a user or a group of users, the do what is called an **action** such as read or update, and the thing is called a **resource** such as a workspace or project. Resources have types which specify what actions are available for its resources, roles (which are collections of actions) and which role is the "owner" role. The "owner" role should have the appropriate actions to administer a resource. When a resource is created a policy with the owner role is automatically created and the creator is added.
```
Comment (s-rubenstein) [inline README.md:5]: This is what specifies user or group above. I figured that seemed more correct, so updated the terms short list

**Comment by s-rubenstein** [comment]
Jenkins retest

**Review by dvoet — APPROVED** [review]

**Comment by s-rubenstein** [comment]
Jenkins retest

**Comment by s-rubenstein** [comment]
Jenkins retest

**Review by gpolumbo-broad — APPROVED** [review]

**Comment by s-rubenstein** [comment]
jenkins retest

**Comment by s-rubenstein** [comment]
Jenkins retest

**Comment by s-rubenstein** [comment]
Jenkins retest

**Pull request merged**